orca-pipelinetemplate/src/test/groovy/com/netflix/spinnaker/orca/pipelinetemplate/v1schema/V1SchemaExecutionGeneratorSpec.groovy

```diff
@@ -114,6 +115,79 @@ class V1SchemaExecutionGeneratorSpec extends Specification {
     []                || ['email-from-configuration@spinnaker.io']
   }
 
+  @Unroll
+  def "should set expected artifacts in execution json"() {
+    given:
+    PipelineTemplate template = getPipelineTemplate()
+    template.configuration.setExpectedArtifacts([createExpectedArtifact('artifact-from-template')])
+
+    TemplateConfiguration configuration = new TemplateConfiguration(
+      pipeline: new PipelineDefinition([application: 'orca', pipelineConfigId: 'pipelineConfigId']),
+      configuration: new TemplateConfiguration.PipelineConfiguration(
+        inherit: inherit,
+        expectedArtifacts: [
+          createExpectedArtifact('artifact-from-configuration')
+        ]
+      )
+    )
+
+    when:
+    def request = new TemplatedPipelineRequest(id: "pipelineConfigId")
+    if (artifactInRequest) {
+      request.setExpectedArtifacts([createExpectedArtifact('artifact-from-request')])
+    }
+    def result = subject.generate(template, configuration, request)
+
+
+    then:
+    result.expectedArtifacts*.id == expectedArtifactIds
+
+    where:
+    inherit               || expectedArtifactIds                                                                || artifactInRequest
+    ['expectedArtifacts'] || ['artifact-from-template', 'artifact-from-configuration']                          || false
+    []                    || ['artifact-from-configuration']                                                    || false
+    []                    || ['artifact-from-request', 'artifact-from-configuration']                           || true
+    ['expectedArtifacts'] || ['artifact-from-template', 'artifact-from-request', 'artifact-from-configuration'] || true
+  }
```

Comment:
Nitpick: `||` is by convention used to separate inputs and expected outputs, so each unroll table should only have one column with these. Use `|` otherwise

Reply:
fixed